gpflow/likelihoods.py

```diff
@@ -583,3 +583,136 @@ def conditional_variance(self, F):
         E_y = tf.matmul(phi, Ys)
         E_y2 = tf.matmul(phi, tf.square(Ys))
         return tf.reshape(E_y2 - tf.square(E_y), tf.shape(F))
+
+
+class MonteCarloLikelihood(Likelihood):
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.num_monte_carlo_points = 100
+        del self.num_gauss_hermite_points
```

Comment:
Maybe at some points in the future we should change this to a single property? E.g. `num_numerical_estimation_points` or something? This should be done after consulting with other people who have looked at quadrature before.